.github/workflows/understack-cli-release.yaml

```diff
@@ -0,0 +1,40 @@
+name: understack-cli-release
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+-understack-cli"
```

Comment:
@skrobul suffix or prefix? other stuff has used a prefix in the repo. I just wanna be consistent.

Reply:
prefix is now allowed as per semver, if prefix is required need to check how to force goreleaser to use that.

Reply:
There example https://goreleaser.com/customization/monorepo/ uses prefixes. Whatever we do I just want consistency. So if we go with suffix then we switch everything to a suffix.